### DISCUSSION TITLE
ZUGFeRD 2.1.1 support

### DISCUSSION THREAD

**Discussion by jr-hoh** [stephanstapel/ZUGFeRD-csharp#341] [General]
Hi everybody,

first of all thanks for this great library and the work that has been put into it. We are planning to use the library for creating the ZUGFeRD invoices in our company.

I have a question regarding the latest [ZUGFeRD version 2.1.1](https://www.ferd-net.de/standards/zugferd-2.1.1/zugferd-2.1.1.html). Is this version supported by the library? I only found some threads regarding XRechnung (https://github.com/stephanstapel/ZUGFeRD-csharp/discussions/174 and https://github.com/stephanstapel/ZUGFeRD-csharp/issues/175) but nothing about the ZUGFeRD versions. The Readme mentions only ZUGFeRD 2.1

Thanks in advance!

**Comment by stephanstapel** [comment]
hmm. The latest ZUGFeRD version is 2.2: https://www.ferd-net.de/standards/zugferd-2.2/zugferd-2.2.html
Concerning supported standards, you find all information in the readme: https://github.com/stephanstapel/ZUGFeRD-csharp

Just let me know if you miss anything.

**Reply by jr-hoh** [reply]
Thanks, that was my bad. I got confused over the different versions.
